optimade/models/links.py

```diff
@@ -61,27 +59,28 @@ class LinksResourceAttributes(Attributes):
 
     link_type: LinkType = Field(
         ...,
-        description="The link type of the represented resource in relation to this implementation. MUST be one of these values: 'child', 'root', 'external', 'providers'.",
+        description="""The link type of the represented resource in relation to this implementation.
+MUST be one of these values: 'child', 'root', 'external', 'providers'.""",
```

Comment:
```suggestion
MUST be one of the values listed in section [Link Types](https://github.com/Materials-Consortia/OPTiMaDe/blob/v1.0.0-rc.2/optimade.rst#link-types).""",
```

Reply:
Why? Do you think we'll add any more link types to make this change worth it? Otherwise this info is not present directly in the openapi schemas

Reply:
It should be as an `Enum`.
And this is simply changing it to the description in the spec.

Reply:
Yeah, it's indeed in the schema through the `Aggregate` `Enum` class.

Reply:
This is the way I've recently been adding "Literal"s :)

Reply:
How do you want to handle this then, if we're not doing internal links? Happy to keep it as is, even if its a bit redundant with the enum?

Reply:
I will do as we do for all the other descriptions, mention a section in the spec, but leave out the link (or add it) and leave it up to #366 to add it properly.

I.e., replicate the spec description.